src/Stream.php

```diff
@@ -193,11 +195,11 @@ public function isWritable()
         $mode = $meta['mode'];
 
         return (
-            strstr($mode, 'x')
-            || strstr($mode, 'w')
-            || strstr($mode, 'c')
-            || strstr($mode, 'a')
-            || strstr($mode, '+')
+            strpos($mode, 'x') !== false
+            || strpos($mode, 'w') !== false
+            || strpos($mode, 'c') !== false
+            || strpos($mode, 'a') !== false
+            || strpos($mode, '+') !== false
```

Comment:
How is this change "better" or an "optimization", exactly? I find it less readable.